### PR TITLE
a few config inspection patches

### PR DESCRIPTION
**Pull request by melvinw**

### CODE REVIEW

File: core/modules/ip_checksum.cc [hunk 1]
```diff
@@ -34,11 +34,20 @@
 #include "../utils/ether.h"
 #include "../utils/ip.h"
 
+const Commands IPChecksum::cmds = {
+    {"get_initial_arg", "EmptyArg", MODULE_CMD_FUNC(&IPChecksum::GetInitialArg),
+     Command::THREAD_SAFE},
+};
+
+CommandResponse IPChecksum::GetInitialArg(const bess::pb::EmptyArg &) {
+  return CommandSuccess();
```
Comment (chris3torek) [inline core/modules/ip_checksum.cc]: As Shinae mentioned elsewhere (offline) we can make this the default. There's a minor danger that someone will then forget to implement it when using a non-trivial initial arg, but I'm not sure if it's worth protecting against this case.

File: core/modules/set_metadata.h [hunk 2]
```diff
@@ -34,11 +34,15 @@
...
+using bess::metadata::mt_offset_t;
 
-typedef struct { uint8_t bytes[kMetadataAttrMaxSize]; } value_t;
-typedef struct { uint8_t bytes[kMetadataAttrMaxSize]; } mask_t;
+typedef struct {
+  uint8_t bytes[kMetadataAttrMaxSize];
+} value_t;
+typedef struct {
+  uint8_t bytes[kMetadataAttrMaxSize];
+} mask_t;
```
Comment (chris3torek) [inline core/modules/set_metadata.h]: (Huh, did clang-format change?)

Comment (melvinw) [reply core/modules/set_metadata.h]: oh, boo it did. didn't even noticed when i scanned the diff earlier

File: core/modules/ip_checksum.cc (hunk 1 above)
Comment (melvinw) [reply core/modules/ip_checksum.cc]: Pushed a fixup with this. Isn't the cleanest, but I'm not sure we can do much better atm

**Pull request merged**